docker-compose.yml

```diff
@@ -130,6 +130,7 @@ services:
   # Fedora repository server
   blazegraph-image: &blazegraph-image
     image: lyrasis/blazegraph:2.1.5
+    platform: linux/amd64
```

Comment:
Can you uncommit this file? I'm not sure this fix actually works well enough to commit yet. We might have to do something similar to OD2, where we rebuild some images